SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialWorldSettings.h

```diff
@@ -2,58 +2,37 @@
 
 #pragma once
 
-#include "LoadBalancing/LayeredLBStrategy.h"
-
-#include "CoreMinimal.h"
-#include "GameFramework/WorldSettings.h"
+#include "LoadBalancing/SpatialMultiWorkerSettings.h"
 #include "SpatialGDKSettings.h"
 #include "Utils/LayerInfo.h"
 
-#include "SpatialWorldSettings.generated.h"
+#include "GameFramework/WorldSettings.h"
+#include "Templates/SubclassOf.h"
 
-class UAbstractLBStrategy;
-class UAbstractLockingPolicy;
+#include "SpatialWorldSettings.generated.h"
 
 UCLASS()
 class SPATIALGDK_API ASpatialWorldSettings : public AWorldSettings
 {
 	GENERATED_BODY()
 
 public:
-	UPROPERTY(EditAnywhere, Config, Category = "Multi-Worker", meta = (EditCondition = "bEnableMultiWorker"))
-	TSubclassOf<UAbstractLBStrategy> DefaultLayerLoadBalanceStrategy;
-
-	UPROPERTY(EditAnywhere, Config, Category = "Multi-Worker", meta = (EditCondition = "bEnableMultiWorker"))
-	TSubclassOf<UAbstractLockingPolicy> DefaultLayerLockingPolicy;
-
-	/**
-	  * Any classes not specified on another layer will be handled by the default layer, but this also gives a way
-	  * to force classes to be on the default layer.
-	  */
-	UPROPERTY(EditAnywhere, Category = "Multi-Worker", meta = (EditCondition = "bEnableMultiWorker"))
-	TSet<TSoftClassPtr<AActor>> ExplicitDefaultActorClasses;
-
-	/** Layer configuration. */
-	UPROPERTY(EditAnywhere, Config, Category = "Multi-Worker", meta = (EditCondition = "bEnableMultiWorker"))
-	TMap<FName, FLayerInfo> WorkerLayers;
+	UPROPERTY(EditAnywhere, Category = "Multi-Worker")
```

Comment:
Nit in the editor this shows as Multi- Worker, an extra space is added after the hyphen

Reply:
I actually want to rename all this usage to be Multiserver instead of Multi-worker so am keen to pushback any name changes for that